python-sdk/langwatch/src/langwatch/evaluations.py

```
@@ -0,0 +1,341 @@
+        data["expected_output"] = expected_output
+    if contexts is not None:
+        data["contexts"] = contexts
+    if expected_contexts is not None:
+        data["expected_contexts"] = expected_contexts
+    if conversation is not None:
+        data["conversation"] = conversation
+    if trace_id is not None:
+        warn("trace_id is deprecated and will be removed in a future version. Future versions of the SDK will not support it. Until that happens, the `trace_id` will be mapped to `deprecated.trace_id` in the data.")
+        data["deprecated.trace_id"] = trace_id
+    if span_id is not None:
+        warn("span_id is deprecated and will be removed in a future version. Future versions of the SDK will not support it. Until that happens, the `span_id` will be mapped to `deprecated.span_id` in the data.")
```

Ruff (0.8.2)

No explicit `stacklevel` keyword argument found

(B028)